pyaedt/modules/SetupTemplates.py

```diff
@@ -48,7 +48,7 @@
 
 HFSSTransient = [("Frequency", "5GHz"), ("MaxDeltaS", 0.02), ("MaximumPasses", 20), ("UseImplicitSolver", True),
                  ("IsEnabled", True), ("MeshLink", meshlink), ("BasisOrder", -1), ("Transient", transienthfss)]
-"""HFSS transient analysis setup properties and default values."""
+"""HFSS transient setup properties and default values."""
```

Comment:
```suggestion
"""HFSS transient analysis setup properties and default values."""
```
I think we should keep analysis.

Reply:
None of the others have "analysis" in them, which is why I removed it (and will leave it out) here.